src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
```

Comment:
2. Error Response: When an exception is thrown in the `throwError` method, the application may return an error response with an HTTP status code of 500 (Internal Server Error). To provide better error handling and responses, consider implementing a global exception handler to handle exceptions consistently across the application.